index.html

```diff
@@ -70,10 +70,13 @@ <h1>GAMERS</h1>
                 <a class="nav-link" href="cart.html">Cart </a>
```

Comment:
Arrow beside the "Home" is not like the design

Reply:
Not done

Reply:
Done